starknet_py/tests/e2e/proxy_check/proxy_test.py

```diff
@@ -1,140 +1,171 @@
+from typing import Optional
+
 import pytest
+from starkware.starknet.public.abi import (
+    get_storage_var_address,
+)
 
 from starknet_py.contract import Contract
+from starknet_py.net.client_errors import ContractNotFoundError
+from starknet_py.proxy_check import ProxyResolutionError, ProxyCheck
+
 
-MAX_FEE = int(1e20)
+async def put_and_get_map_contract(map_contract: Contract, key: int, val: int) -> int:
```

Comment:
I would change this into `assert_map_works_properly` and set&get some custom value in here with an assertion.